app/models/ipeds_hd.rb

```diff
@@ -79,4 +79,8 @@ class IpedsHd < ApplicationRecord
 
   has_many :crosswalk_issue, dependent: :delete_all
   validates :cross, presence: true
+
+  def full_address
+    [addr, city, state, zip].compact.join(', ')
```

Comment:
this is inconsistent  with the implementation in `weam.rb` for address_values and physical_address_values.  I like the `weam` approach better because it puts the view logic (joining into a comma-separated string) into the views.

Reply:
Makes perfect sense. I must have missed this one when I initially cleaned up the other `weam.rb` model. Thanks!